src/app.js

```diff
@@ -0,0 +1,16 @@
+const dotenv = require('dotenv');
```

Comment:
se não for adicionar nenhuma config adicional ao `dotenv` pode passar o `.config()` após o require